Directory.Build.targets

```diff
@@ -244,17 +244,11 @@ Related: https://forum.unity.com/threads/6572-debugger-agent-unable-to-listen-on
   <Target Name="UnitySmokeTestStandalonePlayerIL2CPP" DependsOnTargets="FindUnity" Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'">
     <Error Condition="$(UnityRoot) == ''" Text="Couldn't find Unity."></Error>
 
-    <Message Importance="High" Text="Running smoke test on player."></Message>
-
-    <Exec Command="$(StandaloneExecutablePath) --test smoke" IgnoreExitCode="true" ContinueOnError="true">
-      <Output TaskParameter="ExitCode" PropertyName="SmokeTestExitCode"/>
-    </Exec>
-
-    <Message Condition="'$(StandaloneDataPath)' != ''" Importance="High" Text="Runtime data is found in $(StandaloneDataPath)" />
-    <Exec Condition="$(StandaloneDataPath) != ''" Command="type &quot;$(StandaloneDataPath)/Player.log&quot;"/>
+    <PropertyGroup>
+      <myscript>pwsh "$(RepoRoot)/test/Scripts.Integration.Test/integration-run-smoke-test.ps1" -Smoke -Crash "$(StandaloneExecutablePath)" -AppDataDir "$(StandaloneDataPath)"</myscript>
+    </PropertyGroup>
 
-    <Error Text="Smoke test failed with code $(SmokeTestExitCode)" Condition="'$(SmokeTestExitCode)' != '200'" />
-    <Message Importance="High" Text="Smoke test PASSED" Condition="'$(SmokeTestExitCode)' == '200'" />
+    <Exec Command="$(myscript)" />
```

Comment:
Is there a difference between creating `myscript` inside a property group vs exec command?

Reply:
just `"` vs `quot;` but I've changed it to the latter (i.e. inlined)